### PR TITLE
Content update 14 Feb

### PR DESCRIPTION
**Pull request by gkindavishal**

### CODE REVIEW

**Comment by gkindavishal** [comment]
Hi @5wathi 

Please review and approve - Content update from advocu

File: config/authors/kodeflap.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Pooja Shaji",
+  "bio": "",
```
Comment (5wathi) [inline config/authors/kodeflap.json:3]: Software Developer | Exploring Android Development|Learn and share knowledge

File: config/authors/naingaungluu.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "naingaungluu",
```
Comment (5wathi) [inline config/authors/naingaungluu.json:2]: Name : Naing Aung Luu

File: config/authors/naingaungluu.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+  "bio": "",
```
Comment (5wathi) [inline config/authors/naingaungluu.json:3]: Android Developer at Codigo

File: config/authors/prithviraj2002.json [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "prithviraj_kapil",
+  "bio": "Trying to debug life :)\r\nFlutter/android/compose/mobileapp",
```
Comment (5wathi) [inline config/authors/prithviraj2002.json:3]: Flutter/Android/Compose/Tech

File: config/authors/qamar_safadi.json [hunk 5]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Qamar A. Safadi",
+  "bio": "",
```
Comment (5wathi) [inline config/authors/qamar_safadi.json:3]: As an Android Developer, I possess strong knowledge of Java, but I consider Kotlin to be my primary programming language.

File: config/authors/rohanjsh.json [hunk 6]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Rohan",
+  "bio": "",
```
Comment (5wathi) [inline config/authors/rohanjsh.json:3]: Mobile Developer @GoodCreatorCo

**Review by 5wathi — CHANGES_REQUESTED** [review]

File: config/authors/rohanjsh.json (hunk 6 above)
Comment (gkindavishal) [reply config/authors/rohanjsh.json]: Updated

File: config/authors/qamar_safadi.json (hunk 5 above)
Comment (gkindavishal) [reply config/authors/qamar_safadi.json:3]: Updated

File: config/authors/prithviraj2002.json (hunk 4 above)
Comment (gkindavishal) [reply config/authors/prithviraj2002.json]: Updated

File: config/authors/naingaungluu.json (hunk 3 above)
Comment (gkindavishal) [reply config/authors/naingaungluu.json]: Updated

File: config/authors/naingaungluu.json (hunk 2 above)
Comment (gkindavishal) [reply config/authors/naingaungluu.json]: Updated

File: config/authors/kodeflap.json (hunk 1 above)
Comment (gkindavishal) [reply config/authors/kodeflap.json]: Updated

File: config/authors/amitshekhar.json [hunk 7]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Amit Shekhar",
+  "bio": "",
```
[the remark this answers is not in the document]
Comment (gkindavishal) [reply config/authors/amitshekhar.json]: Updated

File: config/authors/isaacadariku.json [hunk 8]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Isaac Adariku",
+  "bio": "",
```
Comment (5wathi) [inline config/authors/isaacadariku.json:3]: Isaac has spent years with interest in computer intelligence and global activities in the IT world. He is a Lead Software Engineer, Mentor and Community Organizer.

File: config/authors/mustaphaunubi.json [hunk 9]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Mustapha Unubi Momoh",
+  "bio": "",
```
Comment (5wathi) [inline config/authors/mustaphaunubi.json:3]: A Python Developer and Data Scientist passionate about Computer vision and NLP

File: config/authors/prateeksharma.json [hunk 10]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Prateek Sharma",
+  "bio": "",
```
Comment (5wathi) [inline config/authors/prateeksharma.json:3]: Flutter & Android Developer, Founder of Techie Blossom

File: config/authors/tkarmakar27112000.json [hunk 11]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Tanmoy karmakar",
+  "bio": "",
```
Comment (5wathi) [inline config/authors/tkarmakar27112000.json:3]: co-organizer Flutter Kolkata || SDE-II at SaaS Labs || Building Helpwise

File: config/authors/tkko.json [hunk 12]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Tornike",
+  "bio": "",
```
Comment (5wathi) [inline config/authors/tkko.json:3]: Software Engineer/Developer · Experienced in Flutter, Dart, Android, iOS, and Web, with a great sense of UX/UI design.

**Review by 5wathi — CHANGES_REQUESTED** [review]

File: config/android/blogs/bringing-firebase-crashlytics-into-android-studio-with-electric-eles-app-quality-insights-tool-7534616d168.json [hunk 13]
```diff
@@ -0,0 +1,12 @@
+{
+  "expertise": "BEGINNER",
+  "version": 1,
+  "source": "medium",
+  "authorIds": ["sindkarp"],
+  "author": "Priya Sindkar",
+  "title": "Exploring Electric Ele’s App Quality Insights Tool Window",
+  "link": "https://medium.com/novumlogic/bringing-firebase-crashlytics-into-android-studio-with-electric-eles-app-quality-insights-tool-7534616d168",
+  "tags": [
+    "build-tools"
```
[the remark this answers is not in the document]
Comment (gkindavishal) [reply config/android/blogs/bringing-firebase-crashlytics-into-android-studio-with-electric-eles-app-quality-insights-tool-7534616d168.json:10]: @5wathi there's not such tag, are you sure this tag is showing on Advocu?

File: config/authors/arunachermadurai18.json [hunk 14]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Aruna Chermadurai",
+  "bio": "",
```
Comment (5wathi) [inline config/authors/arunachermadurai18.json:3]: Student |  Interested in learning Data Science

**Review by 5wathi — CHANGES_REQUESTED** [review]

**Pull request merged**